test/functional/helpers/spectrum.js

```diff
@@ -276,7 +276,12 @@ const componentWrappers = {
       },
       expectDisabled: createExpectDisabled(selector),
       expectEnabled: createExpectEnabled(selector.find("button")),
-      expectHidden: createExpectHidden(selector.parent().parent())
+      expectHidden: createExpectHidden(
+        selector
+          .parent()
+          .parent()
+          .parent()
+      )
```

Comment:
Can we make this more robust to react-spectrum changes? You can call parent with a selector string.

Reply:
I looked into it but the selector is wrapped in multiple `div` with the same CSS classes